rootfs/bin/is_running

```diff
@@ -4,4 +4,4 @@
 set -e
 
 # check if database is running
-gosu postgres pg_ctl status
+gosu postgres psql -l -t >/dev/null 2>&1
```

Comment:
After this modification, this will done with exit code none zero in the recovery mode.
Is it reasonable?